cli/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@proofgeist/kit",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

don't bump the version. Undo this change


```suggestion
  "version": "0.1.2",
```